doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -786,3 +787,351 @@ JCL      - fd5cc89 based on jdk-11.0.2+7)
+   riscv64-unknown-linux-gnu-gcc (GCC) 9.2.0
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   This is free software; see the source for copying conditions.
+   There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+
+### 4. Get the source for building the JDK
+:rocket: You need to clone the Extensions for OpenJDK for OpenJ9 project, which is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches. 
+
+[1] Run the following command to close the Extensions for OpenJDK:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
+```
+
+[2] Change directory into the cloned repository when the process is complete:
+```
+cd openj9-openjdk-jdk11
+```
+:bulb:
+Cloning this repository can take a while due to the huge size of the OpenJDK project
+
+[3] Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+```
+bash get_source.sh
+```
+
+### 5. Configure the cross-compiling environment
+:rocket:
+When you have all the source files on the host system, run the following configure command to set up the cross-compilation environment:
+```
+bash configure --disable-warnings-as-errors \ 
+               --enable-ddr=no \
+               --with-build-jdk=/<path_to_boot_JDK> \
+               --openjdk-target=riscv64-unknown-linux-gnu \
+               --with-sysroot=/<path_to_your_fedora_mount_directory> \
+               --with-freemarker-jar=/<path_to>/freemarker.jar
+```
+
+:pencil:
+If you want to build an OpenJDK that uses OpenSSL, you must specify `--with-openssl={path_to_library}`
+ where:
+  - `path_to_library` uses an OpenSSL v1 library installed on the mounted Fedora OS image.
+e.g.
+```
+--with-openssl=/<path_to_your_fedora_mount_directory>/usr
+```
+
+You can check the version of OpenSSL on your target system as follows:
+```
+<your_fedora_mount_directory>/usr/bin# file openssl
+openssl: ELF 64-bit LSB executable, UCB RISC-V, version 1 (SYSV), 
+dynamically linked, interpreter /lib/ld-linux-riscv64-lp64d.so.1, 
+for GNU/Linux 4.15.0, BuildID[sha1]=715cfde5a3bdc6ff31b6cc2e449f06df1b3c465a, not stripped
+```
+
+### 6. Cross-build the JDK
+:rocket:
+Now you're ready to cross-build OpenJDK with OpenJ9:
+```
+make all
+```
+
+A binary for the full developer kit (JDK without DDR support) is built and stored in the following directory:
+
+- **build/linux-riscv64-normal-server-release/images/jdk**
+
+Copy the JDK to a specified directory on the mounted Fedora OS image
+e.g.
+```
+cp -r jdk /<path_to>/<your_fedora_mount_directory>/home/<your_user_name>/new_dir
+```
+:bulb:
+Instead of copying the JDK into the Fedora OS image, you can also zip the JDK & upload it onto Fedora via the `scp`  command as follows:
+```
+[root@stage4 home/<your_user_name>/new_dir]# scp  <user>@<the_IP_address_on_your_host_system>:/<path_to>/jdk.zip .
+[root@stage4 home/<your_user_name>/new_dir]# unzip -qq jdk.zip
```

Comment:
Why is the `-q` option repeated?

Reply:
```
$ unzip -help
-q  quiet mode (-qq => quieter)
```
With `-qq` specified, there is no verbose output in the packaging process (it is optional if the user want know what happens to the operation)